src/saving/SaveList.cs

```diff
@@ -207,12 +215,38 @@ public void Refresh()
 
         savesList.QueueFreeChildren();
 
+        allItemsCount = 0;
+        loadedItems = 0;
+
         loadingItem.Visible = true;
         readSavesList = new Task<List<string>>(() => SaveHelper.CreateListOfSaves());
         TaskExecutor.Instance.AddTask(readSavesList);
         EmitSignal(nameof(OnItemsChanged));
     }
 
+    public void Filter(SaveInformation.SaveType saveTypeFilter = SaveInformation.SaveType.Manual, bool showAll = false)
+    {
+        foreach (var item in savesList.GetChildren())
+        {
+            SaveListItem? saveItem = item as SaveListItem;
+
+            if (saveItem != null)
+            {
+                if (showAll)
+                {
+                    ((CanvasItem)item).Show();
```

Comment:
If I'm not mistaken `SaveListItem` inherits from `CanvasItem` so these casts are unnecessary